ament_cmake_gmock/ament_cmake_gmock-extras.cmake

```diff
@@ -21,11 +21,17 @@ macro(_ament_cmake_gmock_find_gmock)
 
     find_package(ament_cmake_test QUIET REQUIRED)
 
+    set(_prefix "${ament_cmake_gmock_DIR}/../../..")
+
     if(NOT GMOCK_FOUND)
       # find gmock include and source folders
-      # fall back to system installed path (i.e. on Ubuntu)
-      set(_search_path_include "/usr/include/gmock")
-      set(_search_path_src "/usr/src/gmock/src")
+      # build the list of locations to search in reverse order
```

Comment:
The location coming from the environment is added conditionally and prepended which is ok. Since the other two are always being consider I think we should add them in order.